src/watchmaker/workers/salt.py

```diff
@@ -266,6 +276,24 @@ def _build_salt_formula(self, extract_dir):
                 filepath=salt_content_file,
                 to_directory=extract_dir
             )
+            if self.salt_content_path:
+                self.log.debug(
+                    'Using salt content path: %s',
+                    self.salt_content_path
+                )
+                try:
+                    salt_content_src = os.sep.join((
```

Comment:
I think this line can be moved out of the try block...

```suggestion
                salt_content_src = os.sep.join((
                	extract_dir, self.salt_content_path))
                try:
                    salt_file_names = os.listdir(salt_content_src)
                    ...
```